contracts/Governance/GovernorBravoDelegate.sol

```diff
@@ -280,6 +283,27 @@ contract GovernorBravoDelegate is GovernorBravoDelegateStorageV2, GovernorBravoE
         emit ProposalQueued(proposalId, eta);
     }
 
+    /**
+     * @notice Whitelists a proposer
+     * @param proposer The address of the proposer to whitelist
+     */
+    function whitelistProposer(address proposer) external {
+        require(msg.sender == address(proposalTimelocks[uint8(ProposalType.NORMAL)]), "GovernorBravo::whitelistProposer: callable only from normal timelock only");
+        whitelistedProposers[proposer] = true;
+
+        emit WhitelistedProposerAdded(proposer);
+    }
+    /**
+     * @notice Removes a whitelisted proposer
+     * @param proposer The address of the proposer to remove from whitelist
+     */
+    function removeWhitelistedProposer(address proposer) external {
+        require(msg.sender == address(proposalTimelocks[uint8(ProposalType.NORMAL)]), "GovernorBravo::removeWhitelistedProposer: callable only from normal timelock only");
```

Comment:
Maybe this should be the critical timelock so that we can be slow to add to the whitelist but quick to remove